### PR TITLE
type annotation for Timestamp and Duration

### PR DESCRIPTION
**Pull request by crockeo**
Issue: https://github.com/python/typeshed/issues/4465

Adds type annotations to Timestamp and Duration methods that were missing return types. All `ToXXXSeconds` types have a return type of `int`, see: [Duration definition](https://github.com/protocolbuffers/protobuf/blob/223e89157a8b8b4f9e57ee4a3cf2165c61372f5d/python/google/protobuf/internal/well_known_types.py#L327-L343) and [Timestamp definition](https://github.com/protocolbuffers/protobuf/blob/223e89157a8b8b4f9e57ee4a3cf2165c61372f5d/python/google/protobuf/internal/well_known_types.py#L217). Duration's `ToTimedelta` is now annotated to return a `timedelta`.

### CODE REVIEW

**Comment by crockeo** [comment]
Sorry for the delay y'all!

@hauntsaninja for sure, it's very little marginal effort.

@Akuli is there any overhead for marking them as `numeric.Real`? I feel like that's the most general acceptable class

**Comment by Akuli** [comment]
The code seems to just pass around whatever values it gets. Is there documentation saying something about this? Do other unrelated parts of the library break if you just pass in floats or e.g. `decimal.Decimal`s?

**Comment by hauntsaninja** [comment]
Thanks for improving this and volunteering to improve it further!

I think I would use `int` everywhere, e.g. https://github.com/protocolbuffers/protobuf/blob/223e89157a8b8b4f9e57ee4a3cf2165c61372f5d/python/google/protobuf/internal/well_known_types.py#L362 seems fairly sketchy if it weren't int. The very structure of (seconds, nanoseconds) and other places in the code seems to indicate ints are what they expect. Documentation isn't explicit, but I think there's a reading of it to suggest int should be used too.

> A Timestamp represents a point in time independent of any time zone or calendar, represented as seconds and fractions of seconds at nanosecond resolution in UTC Epoch time.

**Comment by Akuli** [comment]
I'm quite sure that "nanosecond resolution" means "nanoseconds as int", at least that's how I have seen it being used in documentation of other libraries

File: third_party/2and3/google/protobuf/internal/well_known_types.pyi [hunk 1]
```diff
@@ -16,36 +16,36 @@ class Any_:
 
 class Timestamp:
     def ToJsonString(self) -> str: ...
-    seconds: Any = ...
-    nanos: Any = ...
-    def FromJsonString(self, value: Any) -> None: ...
+    seconds: int = ...
+    nanos: int = ...
+    def FromJsonString(self, value: str) -> None: ...
```
Comment (crockeo) [inline third_party/2and3/google/protobuf/internal/well_known_types.pyi:21]: Should this be six.Text?

**Pull request merged**